tests/BasicSearchTests.FunctionalTests.Core/StartupFunctionalTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+using System;
+using System.Net.Http;
+using BasicSearchTests.FunctionalTests.Core.Models;
+using Newtonsoft.Json;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    public class StartupFunctionalTests
+    {
+        private HttpClient _client;
+        private static string RegistrationBaseUrl = "RegistrationsBaseUrl";
```

Comment:
[https://github.com/NuGet/NuGet.Client/blob/cea464c27a439406c669643e4a6901b56a67a337/src/NuGet.Core/NuGet.Protocol.Core.v3/Constants.cs](https://github.com/NuGet/NuGet.Client/blob/cea464c27a439406c669643e4a6901b56a67a337/src/NuGet.Core/NuGet.Protocol.Core.v3/Constants.cs)

---
In reply to: [81028197](https://github.com/NuGet/NuGet.Services.Metadata/pull/94#discussion_r81028197) [](ancestors = 81028197)